circuit/binary_compare.circom

```diff
@@ -40,57 +40,3 @@ template BinaryCompare(n) {
 }
 
 //------------------------------------------------------------------------------
-
-//
-// given two numbers in `n`-bit binary decomposition (little-endian), we compute
-//
-//   out  :=  (A <= B) ? 1 : 0
-//
-// NOTE: we don't check that the digits are indeed binary;
-//       that's the responsibility of the caller!
-//
-
-template BinaryLessOrEqual(n) {
```

Comment:
I would keep these. It's nice, standalone, not completely trivial (circom development is tricky, I often look back to older code), doesn't hurt anything (the circuit is not changed by adding unused templates), and is used in the testing too (I have tests for these, that breaks if you remove them. Of course you could remove the tests too, but I just don't see the point).

We may possibly need one of these if we need to change to something in the circuit.